osu.Game/Users/Drawables/DrawableAvatar.cs

```diff
@@ -68,7 +68,7 @@ private void openProfile()
             if (!OpenOnClick.Value)
                 return;
 
-            if (user != null)
+            if (user != null && user.Id != 1)
```

Comment:
```suggestion
            if (user?.Id > 1)
```

Reply:
Probably a reasonable comment would also help to understand why should be user Id > 1? If not then what is the fallback?

Reply:
probably making `GuestUser` a static/readonly and referencing its ID directly.